interface/patient_file/summary/disclosure_full.php

```diff
@@ -145,7 +145,7 @@ class='css_button_small iframe' onclick='top.restoreSession()'><span><?php echo
 			<a href='#' class='deletenote css_button_small'
 			id='<?php echo text($iter{id}); ?>' onclick='top.restoreSession()'><span><?php echo xlt('Delete');?></span></a></td>
 			<td class="text" valign='top'><?php echo text($iter{recipient});?>&nbsp;</td>
-			<td class='text' valign='top'><?php if($event[1]=='healthcareoperations'){ echo xlt('health care operations'); } else echo text($event[1]); ?>&nbsp;</td>
+            <td class='text' valign='top'><?php if($event[1]=='healthcareoperations'){ echo xlt('health care operations'); } else echo xlt(text($event[1])); ?>&nbsp;</td>
```

Comment:
the correct way to do this would be the following:
``````
<td class='text' valign='top'><?php if($event[1]=='healthcareoperations'){ echo xlt('health care operations'); } else echo xlt(text($event[1])); ?>&nbsp;</td>
  
            
  Write  Preview

the correct way to do this would be the following:
``````
<td class='text' valign='top'><?php echo text(getListItemTitle('disclosure_type',$event[1])); ?>&nbsp;</td>
``````
Note this function is here: https://github.com/openemr/openemr/blob/master/library/options.inc.php#L3094

So, you also need to require_once the options.inc.php at top of this script for it to work.